ucm2/Allwinner/A64/PinePhone/HiFi.conf

```diff
@@ -0,0 +1,129 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Switch' off"
+		cset "name='Headphone Source Playback Route' DAC"
+		cset "name='Line In Playback Switch' off"
+		cset "name='Line Out Playback Switch' off"
+		cset "name='Line Out Source Playback Route' Mono Differential"
+		cset "name='Mic1 Playback Switch' off"
+		cset "name='Mic2 Playback Switch' off"
+		cset "name='AIF1 DA0 Stereo Playback Route' Stereo"
+		cset "name='AIF3 ADC Source Capture Route' None"
+		cset "name='AIF2 DAC Source Playback Route' AIF2"
+		# AIF2 (Modem) is mono.
+		cset "name='AIF2 DAC Stereo Playback Route' Mix Mono"
+		cset "name='DAC Playback Switch' on"
+		cset "name='ADC Digital DAC Playback Switch' off"
+		cset "name='AIF1 Slot 0 Digital DAC Playback Switch' on"
+		cset "name='AIF2 Digital DAC Playback Switch' off"
+		cset "name='DAC Reversed Playback Switch' on"
```

Comment:
May I assume this routes `DACR` to `MIXL` (and `DACL` to `MIXR`) so `MIX*` are both identival and effectively a mono mix of the DAC outputs?

Reply:
Yes, that is correct.

Reply:
I will document this.

Reply:
Awesome, thanks! (that's a clever use of this output mixer)